client/app/components/bibliothek/index.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Created by Maxi on 24.04.17.
+ */
+
+import React from 'react';
+import style from './style.scss';
+import BookCollection from './../book-collection';
```

Comment:
After my merge you can use app/components/book-collection without using relative paths